RecoEcal/EgammaClusterProducers/python/pippo

```diff
@@ -0,0 +1,76 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Uhm, yes.

On 19.01.2015 13:15, Marco Rovere wrote:
>
> In RecoEcal/EgammaClusterProducers/python/pippo 
> <https://github.com/cms-sw/cmssw/pull/7264#discussion-diff-23156633>:
>
> > @@ -0,0 +1,76 @@
> > +import FWCore.ParameterSet.Config as cms
>
> I guess this file entered by mistake ... ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/7264/files#r23156633>.
>

-- 
Matteo Sani
University of California, San Diego
+41227671577